SentinelOne/cloud_funnel2.0/ingest/parser.yml

```diff
@@ -647,7 +647,7 @@ stages:
       - set:
           file.size: '{{ json_event.message.get("cmdScript.originalSize") }}'
           deepvisibility.script.app_name: '{{ json_event.message.get("cmdScript.applicationName") }}'
-          deepvisibility.script.content: '{{ json_event.message.get("cmdScript.content") }}'
```

Comment:
Can we keep `deepvisibility.script.content` in addition to `action.properties.ScriptBlockText`? 
 the field is used.